scripts/certs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+NAMESPACE="openebs"
+APP_NAME="api-rest"
```

Comment:
This would have to be configurable?

Reply:
for clarification, I created this cert file for testing purposes only until we get cert-manager installed... The plan is to not rely on this script and to use cert-manager to create all the certs.  I am open to discuss and direction on this.

Reply:
Fair enough, though might be worth keeping for devel